pages/home/migrate-to-v2.mdx

```diff
@@ -0,0 +1,41 @@
+# Migration Guide
+
+## Important Notice
+If your project does not have its dependency on `arcade-ai` pinned to the v1 major version, you must pin it to the v1 major version immediately to prevent potential breakages.
+
+## Release Timeline
+Version 2.0.0 will be released on **Tuesday, June 17th during PST morning hours**. Make sure to pin your project's dependency on `arcade-ai` to the v1 major version before this date to avoid any disruptions.
```

Comment:
```suggestion
Version 2.0.0 of the Arcade CLI was released on **Tuesday, June 17th**. Make sure to pin your project's dependency on `arcade-ai` to the v1 major version before this date to avoid any disruptions.
```